src/ConventionalCommitUtilities.js

```diff
@@ -134,7 +147,12 @@ export default class ConventionalCommitUtilities {
     return path.join(pkg.location, CHANGELOG_NAME);
   }
 
-  static changelogPreset(opts) {
-    return opts && opts.changelogPreset ? opts.changelogPreset : "angular";
+  static changelogConfigPath(pkg, opts) {
+    const presetName = opts && opts.changelogPreset ? opts.changelogPreset : "angular";
+    const changelogPreset = allowedPresets.includes(presetName)
+      ? conventionalChangelogPresetLoader.presetLoader(str => str)(presetName)
```

Comment:
The current integration tests seem to be choking on the result of this. Why can't you use the default export of `conventionalChangelogPresetLoader`?